### PR TITLE
added MS club logo as favicon and github logo

### PR DESCRIPTION
**Pull request by sriteja-t**
### Changes:

- added favicon
- added GitHub logo in the social sites
- on clicking the club logo takes user to the home page
- added links to social sites (on clicking on logo)

### Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

#### File(s) Added / Modified

- `/public/favicon`
- `/src/components/navbar.jsx`
- `/src/stylesheets/about.css`
- `/src/routes/About.jsx`
- `/src/images/github.png`

#### Screenshots:

image

### CODE REVIEW

**Comment by KamalDGRT** [comment]
try to squash the commits!

File: src/routes/About.jsx [hunk 1]
```diff
@@ -49,9 +50,11 @@ function About() {
                     </div>
                     <div className='followUs-links'>
                         Follow us on:
-                        <img src={fLogo} alt='Facebook Logo' height={'35px'}></img>
-                        <img src={instagramLogo} alt='Instagram Logo' height={'35px'}></img>
-                        <img src={linkedinLogo} alt='LinkedIn Logo' height={'35px'}></img>
+                        <a href="#" target="_blank"><img src={fLogo} alt='Facebook Logo' height={'35px'}></img></a>
```
Comment (Manoj-Paramsetti) [inline src/routes/About.jsx:53]: You shoud set `rel="noopener"` in <a> tag to ignore CI fails in deployment

File: src/routes/About.jsx [hunk 2]
```diff
@@ -49,9 +50,11 @@ function About() {
...
+                        <a href="https://www.instagram.com/microsoft.sist/" target="_blank"><img src={instagramLogo} alt='Instagram Logo' height={'35px'}></img></a>
```
Comment (Manoj-Paramsetti) [inline src/routes/About.jsx:54]: set `rel="noopener"` in <a> tag

File: src/routes/About.jsx [hunk 3]
```diff
@@ -49,9 +50,11 @@ function About() {
...
+                        <a href="https://www.linkedin.com/company/microsoft-sist/" target="_blank"><img src={linkedinLogo} alt='LinkedIn Logo' height={'35px'}></img></a>
```
Comment (Manoj-Paramsetti) [inline src/routes/About.jsx:55]: set `rel="noopener"` in <a> tag

File: src/routes/About.jsx [hunk 4]
```diff
@@ -49,9 +50,11 @@ function About() {
...
+                        <a href="https://github.com/Microsoft-Club-SIST" target="_blank"><img src={githublogo} alt="Github logo" height={'35px'}></img></a>
```
Comment (Manoj-Paramsetti) [inline src/routes/About.jsx:56]: set `rel="noopener"` in <a> tag

**Review by Manoj-Paramsetti — CHANGES_REQUESTED** [review]

File: src/routes/About.jsx [hunk 5]
```diff
@@ -49,9 +50,11 @@ function About() {
...
+                       
                     </div>
```
Comment (Manoj-Paramsetti) [inline src/routes/About.jsx]: Set `rel="noopener"` in `<a>` tag to ignore CI fails in deployment

**Review by Manoj-Paramsetti — CHANGES_REQUESTED** [review]

File: src/routes/About.jsx (hunk 5 above)
Comment (sriteja-t) [reply src/routes/About.jsx]: fixed 👍🏼

**Review by Manoj-Paramsetti — APPROVED** [review]

**Pull request merged**